app/src/main/res/layout/onboarding_viewpager_item.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You already have a child CL in this, try to use a single CL root for this entire layout and avoid nesting 😁

Reply:
Extract raw values for dimens something like this, similar to strings
![image](https://user-images.githubusercontent.com/84084708/186834687-35cd1b85-f60e-41bf-adaf-f686f67d154e.png)

Reply:
Apply this where necessary in your project